src/core/device.c

```diff
@@ -696,10 +696,7 @@ static void device_update_found_one(Device *d, DeviceFound found, DeviceFound ma
                 d->found = n;
 
                 device_found_changed(d, previous, n);
-        } else
-                /* We aren't running yet, let's apply the new mask to the shadow variable instead, which we'll apply as
-                 * soon as we catch-up with the state. */
-                d->enumerated_found = (d->enumerated_found & ~mask) | (found & mask);
+        }
```

Comment:
hmm, this doesn't look OK to drop. /proc/swaps and /proc/self/mountinfo enumeration code also calls into this indirectly (via device_found_node()) and should have an effect on d->enumerated_found(), or what am I missing?

Reply:
indeed

Reply:
You're not missing anything, I did...